tests/baselines/reference/looseThisTypeInFunctions.js

```diff
@@ -67,7 +68,7 @@ c.explicitVoid = c.explicitThis; // error, 'void' is missing everything
 var o = {
     n: 101,
     explicitThis: function (m) {
-        return m + this.n.length; // ok, this.n: any
+        return m + this.n.length; // error, 'length' does not exist on 'number'

```

Comment:
What about the case that you run the tests, see that it changed, and have to figure out whether or not the change is an improvement? In that case the comment helps you figure out if the test should still pass, and why. 

I've hit several old-ish tests that have accumulated multiple failures -- I couldn't tell if they were *supposed* to fail or even what the test was supposed to show.